core/definitions/src/signature.ts

```diff
@@ -1,22 +1,12 @@
-//TODO this should get properly wrapped and get its own interface so we don't expose implementation
-//  internals and can swap out the implementation if we choose to later. Maybe also rename
-//  recovery to v then (this seems to be the convention at least in EVM land)
-
-import { Signature as SignatureOptionalRecovery } from "@noble/secp256k1";
-
-export class Signature extends SignatureOptionalRecovery {
-  constructor(
-    readonly r: bigint,
-    readonly s: bigint,
-    readonly recovery: number
-  ) {
-    super(r, s, recovery);
-  }
+// Signature represents the secp256k1 signature of a Guardian
+export class Signature {
+  constructor(readonly r: bigint, readonly s: bigint, readonly v: number) {}
```

Comment:
do you need to store these somewhere?

Reply:
nvm, now I remember https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties